File: 백준/Bronze/10987. 모음의 개수/모음의 개수.py
import sys

input = sys.stdin.readline

words = input().strip()
moeums = ['a', 'e', 'i', 'o', 'u']
result = 0

for word in words:
    for mo in moeums:
        if word == mo:
            result += 1
            break

print(result)

File: 백준/Bronze/10987. 모음의 개수/모음의 개수.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String words = br.readLine().trim();
        char[] moeums = {'a', 'e', 'i', 'o', 'u'};
        int result = 0;

        for(int i = 0; i < words.length(); i++){
            for(char mo : moeums){
                if(words.charAt(i) == mo){
                    result++;
                    break;
                }
            }
        }
        System.out.println(result);
    }
}